src/librustdoc/html/render/context.rs

```diff
@@ -623,6 +623,9 @@ impl<'tcx> FormatRenderer<'tcx> for Context<'tcx> {
             let js_dst = self.dst.join("sidebar-items.js");
             let v = format!("initSidebarItems({});", serde_json::to_string(&items).unwrap());
             scx.fs.write(&js_dst, &v)?;
+
+            let js_dst = self.dst.join("top-level-sidebar-items.js");
+            scx.fs.write(&js_dst, "initSidebarItems({});")?;
```

Comment:
I'm a little confused... reading the code above this: if the added code works to add a file at the top level, there should already be a sidebar-items.js at the top level.

Reply:
the `top-level` does not mean the top-level of the generated documentation, but is just an alternative name to distinguish between `initSidebarItems({})` and `initSidebarItems({ /* some items */})`